lib/bolt/util.rb

```diff
@@ -295,6 +295,10 @@ def references?(input)
         end
       end
 
+      def reference?(input)
```

Comment:
This seems like it will only be used in the validator class - I think you can bake it into the `plugin_reference?` method.

Reply:
Ah, I think I had meant for this to replace `Bolt::Plugin.reference?`, I just forgot to change the method calls in `Bolt::Plugin`. I'll just bake it into the validator's `plugin_reference?` method.